src/components/safe-apps/SafeAppList/index.tsx

```diff
@@ -70,18 +86,28 @@ const SafeAppList = ({
           ))}
 
         {/* Flat list filtered by search query */}
-        {safeAppsList.map((safeApp) => (
-          <li key={safeApp.id}>
-            <SafeAppCard
-              safeApp={safeApp}
-              isBookmarked={bookmarkedSafeAppsId?.has(safeApp.id)}
-              onBookmarkSafeApp={onBookmarkSafeApp}
-              removeCustomApp={removeCustomApp}
-              onClickSafeApp={handleSafeAppClick(safeApp)}
-              openPreviewDrawer={openPreviewDrawer}
-            />
-          </li>
-        ))}
+        {safeAppsList.map((safeApp) => {
+          return safeApp.id !== NATIVE_SWAPS_APP_ID ? (
+            <li key={safeApp.id}>
+              <SafeAppCard
+                safeApp={safeApp}
+                isBookmarked={bookmarkedSafeAppsId?.has(safeApp.id)}
+                onBookmarkSafeApp={onBookmarkSafeApp}
+                removeCustomApp={removeCustomApp}
+                onClickSafeApp={handleSafeAppClick(safeApp)}
+                openPreviewDrawer={openPreviewDrawer}
+              />
+            </li>
```

Comment:
Why does this card have to be added into the `safeAppsList`? Why can't it just be rendered outside of the map? Feels very hacky to me.

Reply:
It gets mixed in with the safe apps in the parent component just so that it responds to filtering.
I changed it here to render outside of the map, hopefully it's a bit better

Reply:
I think it's totally fine if it disappears when the list is filtered.